src/components/InjectedComponents/PostDialog.tsx

```diff
@@ -156,9 +157,10 @@ export function PostDialogUI(props: PostDialogUIProps) {
                                         props.onShareToEveryoneChanged(false)
                                     }
                                 }}
-                                label={`A Red Packet with $${formatBalance(BigInt(r.total), r.token?.decimals || 18)} ${
-                                    r.token?.name || 'ETH'
-                                } from ${r.sender.name}`}
+                                label={`A Red Packet with $${formatBalance(
+                                    new BigNumber(r.total),
+                                    r.token?.decimals ?? 18,
```

Comment:
`??` behavior different with `||`.

Reply:
CC @Jack-Works

Reply:
Yes, it's intentional. The original `||` is wrong